### DISCUSSION TITLE
When will the chat file sharing capability be available in a stable production version?

### DISCUSSION THREAD

**Discussion by olandahl** [Azure/communication-ui-library#5096] [General]
In 1.15.0 beta version there's a fileSharing attribute on the ChatComposite component to enable uploading files in a chat. It seems to work fine for our needs but we don't want to use a beta dependency in production. 
It's been around for 5 months, any plans to include it in a stable production version any time soon?

**Comment by Leah-Xia-Microsoft** [comment]
Hi @olandahl, it is in the roadmap and we are evaluating priority and potential changes based on customer feedback. 
You could also submit a feature request in the [feedback forum](https://feedback.azure.com/d365community/forum/81ff6d2b-0c25-ec11-b6e6-000d3a4f0858). Thanks.
